repo/fsrepo/fsrepo.go

```diff
@@ -379,6 +398,61 @@ func (r *FSRepo) openDatastore() error {
 	return nil
 }
 
+var SpecFn = "spec"
```

Comment:
How about `datastore_id`?  The id is not really for the repo but the datastore.  It is a bit more than an id as it also includes the spec but I suppose I can live with that.